### PR TITLE
cdc: add description about claim-check-raw-value

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by 3AceShowHand** [comment]
This feature is planned to be release in v8.4.0

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -535,4 +535,21 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
 }
 ```
 
-`key` 和 `value` 分别存有编码后的大消息，该消息原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
+`key` 和 `value` 分别对应于 Kafka 消息中的同名字段。消费者可以通过解析这两部分的数据，还原大消息的内容。目前，只有 Open-Protocol 编码的 Kafka 消息，有 Key 字段，通过将 Key 和 Value 编码进同一个 JSON 对象，一次性发送完整的消息。对于其余协议，Key 字段总是为空。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:538]:
```suggestion
`key` 和 `value` 分别对应 Kafka 消息中的同名字段。消费者可以通过解析这两部分的数据，还原大消息的内容。只有 Open Protocol 编码的 Kafka 消息包含 `key` 字段。你可以将 `key` 和 `value` 编码到同一个 JSON 对象中，一次性发送完整的消息。对于其他协议，`key` 字段始终为空。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -535,4 +535,21 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
...
+
+#### 只发送 Kafka 消息的 Value 部分到外部存储
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:540]:
```suggestion
### 只发送 `value` 部分到外部存储
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -535,4 +535,21 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
...
+
+从 v8.4.0 开始，支持 `claim-check-raw-value` 参数，该参数默认为 false。当使用非 Open-Protocol 协议时，可以设置为 true，反之则报错。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:542]:
```suggestion
从 v8.4.0 开始，TiCDC 支持仅将 Kafka 消息的 `value` 部分发送到外部存储，该功能仅适用于非 Open Protocol 协议。你可以通过设置 `claim-check-raw-value` 参数控制是否开启该功能，该参数默认值为 `false`。

> **注意：**
>
> 当使用 Open Protocol 协议时，将 `claim-check-raw-value` 参数设置为 `true` 会报错。

当 `claim-check-raw-value` 设置为 `true` 时，changefeed 直接将 Kafka 消息的 `value` 部分发送到外部存储，而无需对 `key` 和 `value` 进行额外的 JSON 序列化。这样可以降低 CPU 开销。此外，消费端可以直接从外部存储读取可直接消费的数据，减少了反序列化过程的开销。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -535,4 +535,21 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
...
+```toml
+protocol = "simple"
+
+[sink.kafka-config.large-message-handle]
+large-message-handle-option = "claim-check"
+claim-check-storage-uri = "s3://claim-check-bucket"
+claim-check-raw-value = true
+```
+
+该参数为 true 时，changefeed 直接发送 Kafka 消息的 Value 部分到外部存储，避免了将 Key 和 Value 进一步执行 JSON 序列化的过程，减少了 CPU 开销。在消费端，可以直接从外部存储读取到可以被直接消费的数据，减少了反序列化过程的开销。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:555]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 5]
```diff
@@ -535,4 +535,25 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
 }
 ```
 
-`key` 和 `value` 分别存有编码后的大消息，该消息原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
+`key` 和 `value` 分别对应 Kafka 消息中的同名字段。消费者可以通过解析这两部分的数据，还原大消息的内容。只有 Open Protocol 编码的 Kafka 消息包含 `key` 字段。将 `key` 和 `value` 编码到同一个 JSON 对象中，一次性发送完整的消息。对于其他协议，`key` 字段始终为空。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:538]:
```suggestion
`key` 和 `value` 分别对应 Kafka 消息中的同名字段。消费者可以通过将 `key` 和 `value` 编码到单个 JSON 对象中，一次性发送完整的消息，从而还原大消息内容。

只有 Open Protocol 编码的 Kafka 消息包含 `key` 字段。对于其他协议，`key` 字段始终为空。
```

Keep it consistent with the English document:

> Consumers can parse the data in these two parts to restore the content of the large message, by encoding the `key` and `value` into one JSON object to deliver a complete message.

File: ticdc/ticdc-sink-to-kafka.md [hunk 6]
```diff
@@ -535,4 +535,25 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
...
+
+### 只发送 `value` 部分到外部存储
+
+从 v8.4.0 开始，TiCDC 支持仅将 Kafka 消息的 `value` 部分发送到外部存储，该功能仅适用于非 Open Protocol 协议。你可以通过设置 `claim-check-raw-value` 参数控制是否开启该功能，该参数默认值为 `false`。
+
+> **注意：**
+>
+> 当使用 Open Protocol 协议时，将 `claim-check-raw-value` 参数设置为 `true` 会报错。
+
+当 `claim-check-raw-value` 设置为 `true` 时，changefeed 直接将 Kafka 消息的 `value` 部分发送到外部存储，而无需对 `key` 和 `value` 进行额外的 JSON 序列化。这样可以降低 CPU 开销。此外，消费端可以直接从外部存储读取可直接消费的数据，减少了反序列化过程的开销。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:548]:
```suggestion
当 `claim-check-raw-value` 设置为 `true` 时，changefeed 会直接将 Kafka 消息的 `value` 部分发送到外部存储，而无需对 `key` 和 `value` 进行额外的 JSON 序列化。这样可以降低 CPU 开销。此外，消费端可以从外部存储读取可直接消费的数据，减少了反序列化过程的开销。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-sink-to-kafka.md [hunk 7]
```diff
@@ -535,4 +535,25 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
 }
 ```
 
-`key` 和 `value` 分别存有编码后的大消息，该消息原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
+`key` 和 `value` 分别对应 Kafka 消息中的同名字段。消费者可以通过解析这两部分的数据，还原大消息的内容。只有 Open Protocol 编码的 Kafka 消息包含 `key` 字段，TiCDC 将 `key` 和 `value` 编码到同一个 JSON 对象中，一次性发送完整的消息。对于其他协议，`key` 字段始终为空。
```
[the remark this answers is not in the document]
Comment (3AceShowHand) [reply ticdc/ticdc-sink-to-kafka.md:538]: 含有 key 字段，但是具体的值，始终是空。

File: ticdc/ticdc-sink-to-kafka.md [hunk 8]
```diff
@@ -535,4 +535,25 @@ Kafka 消费者会收到一条含有大消息在外部存储服务中的地址
 }
 ```
 
-`key` 和 `value` 分别存有编码后的大消息，该消息原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
+`key` 和 `value` 分别对应 Kafka 消息中的同名字段。消费者可以通过解析这两部分的数据，还原大消息的内容。只有 Open Protocol 编码的 Kafka 消息的 `key` 字段包含有效内容，TiCDC 将 `key` 和 `value` 编码到同一个 JSON 对象中，一次性发送完整的消息。对于其他协议，`key` 字段始终为空。
+
+### 只发送 `value` 部分到外部存储
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:540]:
```suggestion
#### 只发送 `value` 部分到外部存储
```

**Pull request merged**